FAQ.md

```diff
@@ -0,0 +1,113 @@
+
+### Will 10101 manage the funds for me?
+
+No, at no time will 10101 hold your bitcoins.
+
+Our application makes a point of respecting and educating users in the principles of self-custody : you hold your own keys, you remain responsible for your wallet and funds, and you retain full control throughout your experience.
+
+That's why you need to make sure you back up your recovery phrase and keep it safe.
+
+### How is it possible to trade directly from my wallet, without a platform?
+
+This prowess is made possible by Discreet Log Contracts.
+
+These are peer-to-peer contacts, involving two parties who lock funds into a specific channel. The release of which is conditional: when the contract expires, it releases the funds in the proportions corresponding to the conditions defined at the outset, using data from an oracle to ensure that the transaction proceeds smoothly, in order to prevent any non-collaborative outcome.
+
+
+DLCs can be seen as an additional layer built on top of the Lightning Network.
+
+If you want to learn more, we produced a series of [blog posts](https://10101.finance/blog/dlc-to-lightning-part-1) on this topic, and another one on the specific case of [trading settlements](https://10101.finance/blog/trade-settlement).
+
+
+### How do USDp and synthetic currencies work?
+
+USDp and synthetic currencies within 10101 are referred to synthetics in finance.
+You will have a USD balance circulating on Lightning, holding nothing but sats and a short position combined.
+
+It's a financial product that hedges you against the volatility of Bitcoin and tracking the value of USD, using only Bitcoin.
+
+If you want to learn more about it we've produced a [blog post](https://10101.finance/blog/synthetic-stable) on this topic.
+
+### What's the point of the recovery phrase?
+
+Your recovery phrase (sometimes called "seed") is the only way to redeem your funds. If you've deleted the app or lost the data on your phone, you will not be able to access your funds without it. We will be unable to assist you either.
+
+You absolutely must save it and keep it in a safe place.
+
+In the case of 10101, this backup will enable you to restore your funds within the app from both your on-chain and lightning wallet, as well as your transaction history and open orders, if you haven't exceeded the time limit* for your futures contracts.
+
+### Do I have to run a node separately to use 10101?
+
+The lightning self-custodial wallet embeds a node directly on your device, so there's no need to run a separate node to use the app.
+
+As for the Bitcoin wallet (On-Chain), it will soon be possible to connect your own node, in order to be independent and sovereign in broadcasting and receiving your transactions from your device.
+
+### How can I fund my Lightning wallet?
+
+To fund your lightning wallet, you'll need to open a channel with 10101, with a minimum deposit size of 60,000 sats and a maximum of 3 million sats.
```

Comment:
Maybe mention that these values are correct at the time of writing and reference the date. Because should be stale soon.

```suggestion
To fund your Lightning wallet, you'll need to open a channel with 10101, with a minimum deposit size of 60,000 sats and a maximum of 3 million sats.
```